ui/models/models.go

```diff
@@ -55,6 +55,9 @@ func (m *Models) RefreshChannels(ctx context.Context) error {
 	index := map[string]*models.Channel{}
 	for i := range channels {
 		index[channels[i].ChannelPoint] = channels[i]
+		if channels[i].ID > 0 {
+			channels[i].Age = m.Info.BlockHeight - uint32(channels[i].ID>>40)
```

Comment:
nice!

Reply:
Not the only change that fixed refresh but the `.Update` method too. I forgot to copy the new field when channels are updated.